build-package.sh

```diff
@@ -1,25 +1,33 @@
 #!/bin/bash
 
 DIR=$(mktemp -d)
+HUGO_VERSION=0.17
+
 cd $DIR
 
 # Install awscli python lib + deps in buildroot
-pip install awscli -t .
+pip install pyyaml awscli -t .
+
 # Fetch aws cli wrapper from github
-wget https://raw.githubusercontent.com/aws/aws-cli/1.9.15/bin/aws \
+wget https://raw.githubusercontent.com/aws/aws-cli/develop/bin/aws \
```

Comment:
I updated this to use the latest release instead. Seems like they were using develop as their main branch, and I just went for that one!